WordPress/WordPressUITests/Tests/EditorGutenbergTests.swift

```diff
@@ -15,7 +15,9 @@ class EditorGutenbergTests: XCTestCase {
 
     override func tearDownWithError() throws {
         takeScreenshotOfFailedTest()
+
         if editorScreen != nil && !TabNavComponent.isVisible() {
+            editorScreen.dismissBlocksPickerIfNeeded()
             EditorFlow.returnToMainEditorScreen()
             editorScreen.closeEditor()
```

Comment:
I thought about moving the `dismissBlockPickerIfNeeded() call in the `closeEditor()` method, but it wouldn't have worked because we need to call `EditorFlow.returnToMainEditorScreen()` first. `EditorFlow` exists the UI tests target but not in the UITestsFoundation framework, so I decided to leave it as it is.